authorization/jsonld-contexts/authorization.jsonld

```diff
@@ -3,9 +3,6 @@
         "Client": "https://ontology.eglobalmark.com/authorization#Client",
         "Group": "https://ontology.eglobalmark.com/authorization#Group",
         "User": "https://ontology.eglobalmark.com/authorization#User",
-        "canAdmin": "https://ontology.eglobalmark.com/authorization#canAdmin",
-        "canRead": "https://ontology.eglobalmark.com/authorization#canRead",
-        "canWrite": "https://ontology.eglobalmark.com/authorization#canWrite",
```

Comment:
removing terms that will be unused after the upgrade will break all existings installations before the upgrade (and the one that have not yet migrated after the upgrade)